### PR TITLE
hoc overview: Show Oceans for non-en users

### PR DESCRIPTION
**Pull request by breville**
Now that the Oceans for AI tutorial has been localised into a bunch of languages, show it for non-en users at https://code.org/hourofcode/overview.

### before, en

Screenshot 2020-03-11 15 41 32

### before, non-en

Screenshot 2020-03-11 16 15 45

### after, en

Screenshot 2020-03-11 15 42 51

### after, non-en

Screenshot 2020-03-11 16 06 10

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml [hunk 1]
```diff
@@ -30,51 +30,38 @@ social:
...
+        %img.tutorial-tile-img{src: "/images/fill-485x235/oceans/oceans-overview.png"}
+      .tutorial-info
+        %span
+          %h3.tutorial-info-h=hoc_s(:hoc_overview_oceans_tutorial_header)
+          .smalltext=hoc_s(:hoc_overview_oceans_tutorial_desc)
+        %span
+          %a{href: "/oceans"}
+            %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
+          .tutorial-info-guide
+            %a{href: "https://curriculum.code.org/hoc/plugged/9/"}
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/hourofcode/overview.haml:63]: Is this translated now too?

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml [hunk 2]
```diff
@@ -30,51 +30,38 @@ social:
...
+  .col-50
+    .tutorial-tile
+      %a{href: "/oceans"}
+        %img.tutorial-tile-img{src: "/images/fill-485x235/oceans/oceans-overview.png"}
+      .tutorial-info
+        %span
+          %h3.tutorial-info-h=hoc_s(:hoc_overview_oceans_tutorial_header)
+          .smalltext=hoc_s(:hoc_overview_oceans_tutorial_desc)
+        %span
+          %a{href: "/oceans"}
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/hourofcode/overview.haml:60]: /oceans is translatable, but most of it is not translated (looks like the try now button and MS sponsorship are) on prod right now

**Review by Erin007 — APPROVED** [review]
Looks good! A couple questions about linking off to non-translated content, but I assume that's being tracked and in the i18n roadmap somewhere down the line.

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml (hunk 1 above)
Comment (breville) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml:63]: Good question.  I'm not sure that we translate these curriculum.code.org pages.  It doesn't seem to be translated for Dance Party 2019 either.

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml (hunk 2 above)
Comment (breville) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml:60]: I'm told we are about to get a set of translations for Oceans-related content which includes strings on the site, so it should be incoming, but I will double-check.

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml (hunk 1 above)
Comment (Erin007) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml:63]: We can translate curriculum.code.org pages, for example: https://curriculum.code.org/it-it/hoc/plugged/9/ but it looks like these lesson plans are mostly not translated.  Maybe these lesson plan strings are also in the upcoming incoming translations?

**Review by islemaster — APPROVED** [review]
:joy: Tried to review this without whitespace changes, and realized it doesn't make much sense because whitespace is important in HAML!  LGTM.

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml (hunk 1 above)
Comment (breville) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml:63]: Apparently so!

**Pull request merged**